packages/backend/package.json

```diff
@@ -13,7 +13,7 @@
     "deployCORS": "gsutil cors set ../backend/firebase-storage-cors.json gs://metabook-system.appspot.com",
     "deployFirestoreConfig": "firebase deploy --only firestore",
     "fetchFirestoreIndexes": "firebase firestore:indexes > ../backend/firestore.indexes.json",
-    "test": "jest"
+    "test": "jest --runInBand"
```

Comment:
Ah: I'd actually meant to add this a few days ago anyway! My new Firestore `DatabaseBackend` test's implementations need this anyway. Great!